Simulation/interface/CosmicGun.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CosmicGun_H
+#define CosmicGun_H
+
+/** \class CosmicGun
+ *
+ * Generates single particle gun in HepMC format
+ * Julia Yarba 12/2005 
+ ***************************************/
+
+#include "FWCore/PluginManager/interface/ModuleDef.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "gemsw/Simulation/interface/BaseFlatGunProducer.h"
```

Comment:
why not use IOMC/ParticleGuns/interface/BaseFlatGunProducer.h?

Reply:
It's not compile-able if we use the one from IOMC/ParticleGuns.
https://github.com/cms-sw/cmssw/blob/master/IOMC/ParticleGuns/BuildFile.xml#L8
It's not a library...

Reply:
I will try to figure out